src/configurations/destinations/tiktok_ads/schema.json

```
@@ -806,6 +830,52 @@
                 }
               ]
             }
+          },
+          "shopify": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "provider": {
+                  "type": "string",
+                  "enum": ["custom", "ketch", "oneTrust"],
```

Can we add iubenda also? Ref: https://github.com/rudderlabs/rudder-integrations-config/pull/1795
```suggestion
                  "enum": ["custom", "iubenda", "ketch", "oneTrust"],
```

---

@mohamedkhattab, it's better to wait for this PR to be merged. You can pull the latest changes from upstream and make the necessary changes.

@yashasvibajpai, what's the tentative date we're planning to merge it?

---

This can be a part of post 1st code freeze release.